lib: uicc_lwm2m: Add UICC LwM2M library (#16855)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

    * Added a call to the :c:func:`nrf_cloud_print_details` function and removed redundant logging.

> * :ref:`uicc_lwm2m_sample` sample:
> 
>     * Added the `uicc_lwm2m_sample` sample.

Reviewer: ```suggestion
* :ref:`uicc_lwm2m_sample` sample:

  * Added the :ref:`uicc_lwm2m_sample` sample.
```

  * Added sysbuild configuration files.

> * :ref:`uicc_lwm2m_sample` sample:
> 
>   * Added the :ref:`uicc_lwm2m_sample` sample.

Reviewer 2: ```suggestion
* Added the :ref:`uicc_lwm2m_sample` sample.
```
Move this right under the Cellular samples heading.

     The :ref:`at_parser_readme` is a library that parses AT command responses, notifications, and events.
     Compared to the deprecated :ref:`at_cmd_parser_readme` library, it does not allocate memory dynamically and has a smaller footprint.
     For more information on how to transition from the :ref:`at_cmd_parser_readme` library to the :ref:`at_parser_readme` library, see the :ref:`migration guide <migration_2.8_recommended>`.
   * The :ref:`uicc_lwm2m_readme` library.

>      The :ref:`uicc_lwm2m_readme` is a library that read LwM2M bootstrap configuration from SIM.

Reviewer: Seems this is not fixed.

>      The :ref:`uicc_lwm2m_readme` is a library that read LwM2M bootstrap configuration from SIM.

Author: Right, now also fixed the `read` -> `reads`.

     The :ref:`at_parser_readme` is a library that parses AT command responses, notifications, and events.
     Compared to the deprecated :ref:`at_cmd_parser_readme` library, it does not allocate memory dynamically and has a smaller footprint.
     For more information on how to transition from the :ref:`at_cmd_parser_readme` library to the :ref:`at_parser_readme` library, see the :ref:`migration guide <migration_2.8_recommended>`.
   * The :ref:`lib_uicc_lwm2m` library.

>      The :ref:`lib_uicc_lwm2m` is a library that reads LwM2M bootstrap configuration from SIM.

Reviewer 2: ```suggestion
     This library reads the LwM2M bootstrap configuration from SIM.
```
